scripts/release.py

```diff
@@ -32,7 +32,16 @@
 
 JIRA_RE = re.compile(r"\b[A-Z]{2,}-\d+\b")
 PR_BRANCH_RE = re.compile(r"remotes/edx/pr/(\d+)")
-PROJECT_ROOT = path(__file__).abspath().dirname()
+
+
+def project_root():
```

Comment:
Why change this code? Did the old code work? Though I don't see how, it seem like the old code produced "edx-platform/scripts", and this code will produce "edx-platform"?